101-app-function/azuredeploy.json

```diff
@@ -0,0 +1,79 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "webAppName": {
+            "type": "string",
+            "metadata": {
+                "description": "The name of you Web Site."
+            }
+        },
+        "location": {
+        "type": "string",
+        "defaultValue": "[resourceGroup().location]",
+        "metadata": {
+                "description": "Location for all resources."
+            }
+        }
+    },
+     "variables": {
+        "hostingPlanName": "[concat('hp', uniqueString(resourceGroup().name))]",
+        "storageAccountName": "[concat('sa',  uniqueString(resourceGroup().name))]"
```

Comment:
this needs to be globally unique

Reply:
Hey Brian, thanks for your comments.

So, in order to share the resources in the same group, changing to create the uniqueString from name to id I think that we will have a globally unique name, right?